impeller/compiler/shader_lib/impeller/texture.glsl

```diff
@@ -19,6 +19,14 @@ vec4 IPSample(sampler2D texture_sampler, vec2 coords, float y_coord_scale) {
   return texture(texture_sampler, coords);
 }
 
+/// Flip coordinates if If `y_coord_scale` < 0.0.
+vec2 IPRemapCoords(vec2 coords, float y_coord_scale) {
```

Comment:
Not sure if this is actually that useful. Decreases arthimetic unit usage, increases varying or load/store unit usage

Reply:
this ends up with more effect on shaders like gaussian blur that sample N times per fragment